docs/advanced/zk_intuition.md

```diff
@@ -146,7 +146,7 @@ version 1.4.0.
 [example ecrecover circuit]:
   https://github.com/matter-labs/sync_vm/blob/683ade0bbb445f3e2ceb82dd3f4346a0c5d16a78/src/glue/ecrecover_circuit/mod.rs#L157
 [zksync core witness]:
-  https://github.com/matter-labs/zksync-2-dev/blob/847e7e099175052ccdcad90ea8b3f92c443cc8cd/core/bin/zksync_core/src/witness_generator/mod.rs#L18
+  https://github.com/matter-labs/zksync-era/blob/main/core/lib/zksync_core/src/witness_generator/mod.rs
 [separate witness binary]:
   https://github.com/matter-labs/zksync-2-dev/blob/847e7e099175052ccdcad90ea8b3f92c443cc8cd/core/bin/witness_generator/src/main.rs#L37
```

Comment:
It appears that there are other broken links in this file. Could you please go through the file and fix other links as well? Thanks in advance!

Reply:
> It appears that there are other broken links in this file. Could you please go through the file and fix other links as well? Thanks in advance!

Can you give me some advice? I seem to be unable to find where another error link should point to.

Reply:
> > It appears that there are other broken links in this file. Could you please go through the file and fix other links as well? Thanks in advance!
> 
> Can you give me some advice? I seem to be unable to find where another error link should point to.

separate witness binary

Reply:
I think it resides [here](https://github.com/matter-labs/zksync-era/blob/main/prover/witness_generator/src/main.rs) right now.

Reply:
> I think it resides [here](https://github.com/matter-labs/zksync-era/blob/main/prover/witness_generator/src/main.rs) right now.

Thank you, I have submitted.

Reply:
Hi, can you merge and close this PR? @popzxc